app/api/dao/user.py

```diff
@@ -235,77 +235,33 @@ def update_user_profile(user_id: int, data: Dict[str, str]):
                     messages.USER_USES_A_USERNAME_THAT_ALREADY_EXISTS,
                     HTTPStatus.BAD_REQUEST,
                 )
-
             user.username = username
 
-        if "name" in data and data["name"]:
-            user.name = data["name"]
+        user.name = data.get("name", None)
 
-        if "bio" in data:
-            if data["bio"]:
-                user.bio = data["bio"]
-            else:
-                user.bio = None
+        user.bio = data.get("bio", None)
 
-        if "location" in data:
-            if data["location"]:
-                user.location = data["location"]
-            else:
-                user.location = None
+        user.location = data.get("location", None)
 
-        if "occupation" in data:
-            if data["occupation"]:
-                user.occupation = data["occupation"]
-            else:
-                user.occupation = None
+        user.occupation = data.get("occupation", None) or None
 
-        if "organization" in data:
-            if data["organization"]:
-                user.organization = data["organization"]
-            else:
-                user.organization = None
+        user.organization = data.get("organization", None) or None
```

Comment:
Same as above, same for other cases as well
```suggestion
        user.organization = data.get("organization")
```

Reply:
As I have told earlier, on removing none in organization and occupation will fail test cases.